src/keywords.js

```diff
@@ -10,6 +10,13 @@ export syntax galat = function (ctx) {
     return #`false`;
 }
 
+
+// isNaN
+export syntax kya_galat = function (ctx) {
+    let params = ctx.next().value
+    return #`isNaN ${params}`;
+}
+
 // if
```

Comment:
`KnH` is better though.

Anybody else?
Should I change **kya_galat** to **KnH** ?